### PR TITLE
feat(nextjs live avatars): convert to 0.18

### PR DESCRIPTION
**Pull request by FlowFlorent**
- Adds `initialPresence`
- Removes deprecated `.toArray()`

### CODE REVIEW

File: examples/nextjs-live-avatars/pages/index.tsx [hunk 1]
```diff
@@ -38,7 +38,7 @@ export default function Page() {
   const roomId = useOverrideRoomId("nextjs-live-avatars");
 
   return (
-    <RoomProvider id={roomId}>
+    <RoomProvider id={roomId} initialPresence={{}}>
```
Comment (FlowFlorent) [inline examples/nextjs-live-avatars/pages/index.tsx:41]: @nvie Is this something we'd like to enforce? Should we default `initialPresence` to `{}` if not set?

**Review by nvie — APPROVED** [review]
Awesome! Nice work, @FlowFlorent — thanks for helping to get these converted! 🙌

File: examples/nextjs-live-avatars/package.json [hunk 2]
```diff
@@ -8,9 +8,9 @@
     "start": "next start"
   },
   "dependencies": {
-    "@liveblocks/client": "^0.17.11",
-    "@liveblocks/node": "^0.17.11",
-    "@liveblocks/react": "^0.17.11",
+    "@liveblocks/client": "^0.18.0-beta3",
+    "@liveblocks/node": "^0.18.0-beta3",
+    "@liveblocks/react": "^0.18.0-beta3",
```
Comment (nvie) [inline examples/nextjs-live-avatars/package.json:13]: I just published 0.18.0-beta4, which hopefully is the final version that we can launch as 0.18.0 next week. Could you upgrade once more? Sorry 'bout the hassle!

File: examples/nextjs-live-avatars/pages/index.tsx (hunk 1 above)
Comment (FlowFlorent) [reply examples/nextjs-live-avatars/pages/index.tsx:41]: Ah perfect, thank you so much for this explanation, it makes perfect sense!

**Pull request merged**